### ISSUE TITLE
Test case class not matching filename is deprecated, Class name was 'ReedSolomonTest', expected 'ReedSolomonCodecTest'

### ISSUE THREAD

**Issue by williamdes** [Bacon/BaconQrCode#97]
https://salsa.debian.org/php-team/pear/php-bacon-baconqrcode/-/jobs/2439442#L984
Warning:       Test case class not matching filename is deprecated
               in /builds/php-team/pear/php-bacon-baconqrcode/debian/output/source_dir/test/Common/ReedSolomonCodecTest.php
               Class name was 'ReedSolomonTest', expected 'ReedSolomonCodecTest'

https://github.com/Bacon/BaconQrCode/blob/0069435e2a01a57193b25790f105a5d3168653c1/test/Common/ReedSolomonCodecTest.php#L10

**Comment by williamdes** [comment]
Hi @DASPRiD 
Could you push a fix for this one please ?

**Comment by DASPRiD** [comment]
Fix merged.

**Comment by williamdes** [comment]
> Fix merged.

Awesome, thank you!
Let me know when you make a release for it

**Comment by DASPRiD** [comment]
Will do, currently waiting on another PR before making a new release.
